components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -660,7 +660,7 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
     public async getLoggedInUser(ctx: TraceContext): Promise<User> {
         await this.doUpdateUser();
         const user = this.checkUser("getLoggedInUser");
-        return this.cleanUser(user);
```

Comment:
I have moved and sanitized the translation between DBUSer <-> User in user-db-impl